pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -3860,106 +3860,2824 @@ func Test_server_StreamInsert(t *testing.T) {
+							})
+						return err
+					}(),
+				},
+			}
+		}(),
+		func() test {
+			insertNum := 100
+			req := genMultiInsertReq(Float, vector.Gaussian, insertNum, f32VecDim+1, nil)
+
+			return test{
+				name: "Equivalence Class Testing case 3.5: Fail to MultiInsert 100 vector with all vector with different dimension (vector type is uint8)",
+				args: args{
+					ctx:  ctx,
+					reqs: req,
+				},
+				fields: fields{
+					name:    name,
+					ip:      ip,
+					svcCfg:  defaultF32SvcCfg,
+					svcOpts: defaultSvcOpts,
+				},
+				want: want{
+					err: func() error {
+						err := errors.ErrIncompatibleDimensionSize(len(req.Requests[0].Vector.Vector), intVecDim)
+						err = status.WrapWithInvalidArgument("MultiInsert API Incompatible Dimension Size detected",
+							err,
+							&errdetails.RequestInfo{
+								RequestId:   req.Requests[0].Vector.Id,
+								ServingData: errdetails.Serialize(req),
+							},
+							&errdetails.BadRequest{
+								FieldViolations: []*errdetails.BadRequestFieldViolation{
+									{
+										Field:       "vector dimension size",
+										Description: err.Error(),
+									},
+								},
+							},
+							&errdetails.ResourceInfo{
+								ResourceType: ngtResourceType + "/ngt.MultiInsert",
+								ResourceName: fmt.Sprintf("%s: %s(%s)", apiName, name, ip),
+							})
+						return err
+					}(),
+				},
+			}
+		}(),
+		func() test {
+			insertNum := 100
+			req := genMultiInsertReq(Float, vector.Gaussian, insertNum, f32VecDim+1, nil)
+
+			return test{
+				name: "Equivalence Class Testing case 3.6: Fail to MultiInsert 100 vector with all vector with different dimension (vector type is float32)",
+				args: args{
+					ctx:  ctx,
+					reqs: req,
+				},
+				fields: fields{
+					name:    name,
+					ip:      ip,
+					svcCfg:  defaultF32SvcCfg,
+					svcOpts: defaultSvcOpts,
+				},
+				want: want{
+					err: func() error {
+						err := errors.ErrIncompatibleDimensionSize(len(req.Requests[0].Vector.Vector), intVecDim)
+						err = status.WrapWithInvalidArgument("MultiInsert API Incompatible Dimension Size detected",
+							err,
+							&errdetails.RequestInfo{
+								RequestId:   req.Requests[0].Vector.Id,
+								ServingData: errdetails.Serialize(req),
+							},
+							&errdetails.BadRequest{
+								FieldViolations: []*errdetails.BadRequestFieldViolation{
+									{
+										Field:       "vector dimension size",
+										Description: err.Error(),
+									},
+								},
+							},
+							&errdetails.ResourceInfo{
+								ResourceType: ngtResourceType + "/ngt.MultiInsert",
+								ResourceName: fmt.Sprintf("%s: %s(%s)", apiName, name, ip),
+							})
+						return err
+					}(),
+				},
+			}
+		}(),
+		func() test {
+			insertNum := 100
+			return test{
+				name: "Boundary Value Testing case 1.1: Success to MultiInsert with 0 value vector (vector type is uint8)",
+				args: args{
+					ctx:  ctx,
+					reqs: genSameVecMultiInsertReq(insertNum, []float32{0, 0, 0}, nil),
```

Comment:
```suggestion
					reqs: genSameVecMultiInsertReq(insertNum, [intVecDim]float32, nil),
```

Reply:
It would be better to show the reason set 3 variables in a slice.

Reply:
we cannot define like your suggestion.
I will refactor it to 
```suggestion
					reqs: genSameVecMultiInsertReq(insertNum, make([]float32, f32VecDim), nil),
```

Reply:
or should I create a helper function to fix it?

Reply:
If this function gives us any good merit, it would be better.
Either is fine..

Reply:
yes, I think we can also use the helper function for different value tests (like https://github.com/vdaas/vald/pull/1612#discussion_r879977142). 
I will fix this comment by defining the helper function for it.

Reply:
fixed :)